nixos/lib/make-options-doc/default.nix

```diff
@@ -39,12 +39,17 @@
 # allow docbook option docs if `true`. only markdown documentation is allowed when set to
 # `false`, and a different renderer may be used with different bugs and performance
 # characteristics but (hopefully) indistinguishable output.
-, allowDocBook ? true
+# deprecated since 23.11.
+# TODO remove in a while.
```

Comment:
Would it be reasonable to set this to be removed before the next major release (24.05)? On a more general note, is there a fixed format we can use to explicitly mark TODOs for specific releases?